File: backend/AudApi/AudApi/DTOs/VerEmpleadoDTO.cs
using AudApi.Models;

namespace AudApi.DTOs
{
    public class VerEmpleadoDTO
    {
        public int EmpleadoId { get; set; }

        public string? Nombres { get; set; }

        public string? Apellidos { get; set; }

        public string? Identidad { get; set; }

        public DateOnly? FechaContratacion { get; set; }

        public string? Telefono { get; set; }

        public string? Correo { get; set; }

        public int? NivelEducativoId { get; set; }
        public string NivelEducativoNombre { get; set; }
        public int? CargoId { get; set; }
        
        public string CargoNombre { get; set; }

        

        public List<EmpleadoUsuarioDTO> Usuarios { get; set; } = new List<EmpleadoUsuarioDTO>();
    }

    public class EmpleadoUsuarioDTO
    {
        public int Id { get; set; }
        public string Username { get; set; }

        }
}

File: frontend/mainmenu/src/app/ver-empleado.dto.ts
// models/ver-empleado.dto.ts (puedes usar otra ruta si preferís)
export interface VerEmpleadoDTO {
    empleadoId: number;
    nombres: string;
    apellidos: string;
    identidad: string;
    fechaContratacion: string;
    telefono: string;
    correo: string;
    nivelEducativoId: number;
    nivelEducativoNombre: string;
    cargoId: number;
    cargoNombre: string;
    usuarios: { id: number; username: string }[];
  }
  